recipes/ignition-math/all/conanfile.py

```diff
@@ -77,11 +78,11 @@ def build_requirements(self):
 
     def source(self):
         tools.get(**self.conan_data["sources"][self.version])
```

Comment:
```suggestion
        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
```
So you can remove the rename step